schema/project-level/project-schema.json

```diff
@@ -2336,6 +2347,58 @@
         }
       },
       "minProperties": 1
+    },
+    "Benefit": {
+      "title": "Benefit",
+      "description": "A benefit or intended benefit of the project.",
+      "type": "object",
+      "properties": {
+        "title": {
+          "title": "Title",
+          "description": "A short, locally unique title for the benefit.",
+          "type": "string"
+        },
+        "description": {
+          "title": "Description",
+          "description": "A short description of the benefit.",
+          "type": "string"
+        },
+        "beneficiaries": {
+          "title": "Beneficiaries",
+          "description": "Whom the benefit will aid.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Beneficiary"
+          },
+          "wholeListMerge": true,
```

Comment:
Just remembered that `wholeListMerge` isn't needed in OC4IDS because there is no releases/records/merging mechanism.

```suggestion
```

Reply:
As commented on in another PR where you've suggested removing this. This is needed to make the tests pass where there are array of objects without required `id` fields. My preference is to keep this rather than add in a lot of extra id fields...

Reply:
I've asked James what to do in https://github.com/open-contracting/infrastructure/pull/434#discussion_r1403700571

Reply:
We'll need to add an exception to the tests.

Reply:
@jpmckinney are you OK to do that or shall I?